addons/test_mail/tests/test_mail_followers.py

```diff
@@ -122,6 +122,14 @@ def test_followers_multiple_subscription_update(self):
         test_record.message_subscribe(partner_ids=[self.user_employee.partner_id.id], subtype_ids=[self.mt_cl_def.id, self.mt_al_def.id])
         self.assertEqual(follower.subtype_ids, self.mt_cl_def | self.mt_al_def)
 
+    def test_followers_inverse_message_partner(self):
```

Comment:
I would use the ``@users`` decorator, always better to test using standard users instead of admin :)